deploy/marathon/resources/connected-car.json

```diff
@@ -8,30 +8,10 @@
           "image": "thinkmorestupidless/connected-car-lagom-impl:1.0.27",
           "network": "BRIDGE",
           "portMappings": [
-            {
-              "containerPort": 9000,
-              "hostPort": 0,
-              "servicePort": 10204,
-              "protocol": "tcp",
-              "name": "http"
-            },
-            {
-              "containerPort": 2551,
-              "hostPort": 0,
-              "servicePort": 10205,
-              "protocol": "udp,tcp",
-              "name": "remoting"
-            },
-            {
-              "containerPort": 19999,
-              "hostPort": 0,
-              "servicePort": 10206,
-              "protocol": "tcp",
-              "name": "akkamgmthttp"
-            }
+            { "containerPort": 0, "servicePort": 0, "name": "http" },
+            { "containerPort": 0, "servicePort": 0, "name": "akkaremote" },
+            { "containerPort": 0, "servicePort": 0, "name": "akkamgmthttp" }
```

Comment:
marathon part of naming ports, plus `0` lets DC/OS assign ports which is the most portable across network configuration types